build_all.sh

```diff
@@ -69,6 +75,9 @@ else
 #jp4/5
     cd $SRCS/$KERNEL_DIR
     make ARCH=arm64 O=$TEGRA_KERNEL_OUT tegra_defconfig
+    if [[ "$DEVDBG" == "1" ]]; then
+        scripts/config --file $TEGRA_KERNEL_OUT/.config --enable DYNAMIC_DEBUG
```

Comment:
What will happen for JP4 here? (not mandatory as we stop support but better not to break it too fast)

Reply:
It won't break JP4, as this flag anyway present in JP4 and enabled by default.